ios/Classes/FlutterSoundPlugin.m

```diff
@@ -357,8 +357,9 @@ - (void)resumePlayer:(FlutterResult)result {
 
 - (void)seekToPlayer:(nonnull NSNumber*) time result: (FlutterResult)result {
   if (audioPlayer) {
-    audioPlayer.currentTime = [time doubleValue];
-    result(time);
+      audioPlayer.currentTime = [time doubleValue] / 1000;
+      [self updateProgress:nil];
```

Comment:
Is this line needed? `[self updateProgress:nil]`?

Reply:
This line isn't strictly necessary. After a bit more investigation, this can be handled by the host application's logic. However, having this line provides a confirmation that the seekToPlayer function correctly updated the AVAudioPlayer's currentTime. I'm open to removing or leaving it, either way.

Reply:
Alright! Let's keep this update for now.